test/app.test.ts

```diff
@@ -86,9 +86,7 @@ describe('JavaScript', () => {
   });
 
   test('card numbers are detected but ignored numbers are not included', async () => {
-    expect(await scanFiles({ files: '**/test/fixtures/javascript/*.{ts,js}' , ignoreNumbers: ["5555554240233167"] })).toEqual([
-      'test/fixtures/javascript/bad.js'
-    ]);
+    expect(await scanFiles({ files: '**/test/fixtures/javascript/*.{ts,js}' , ignoreNumbers: ["5555554240233167"] })).toEqual([]);
```

Comment:
@darrenpicard25 I'm not sure how this was supposed to work, but if we're ignoring the number, I don't think it should be included in the results array.

Reply:
I'm trying to remember about this but it feels so long ago. What you are saying makes sense

Reply:
just for my sanity can you add a test where we are ignoring some cardNumbers but not all and so the file gets flagged